### PR TITLE
feat: VPL simulation files fix: lab 2 drills feat: first two courses latex slides feat: add the opensource toolchain Dockerfile

### PR DESCRIPTION
**Pull request by sdcioc**
- added the VPL simulation example files and cripts
 - fix the name of the tasks and xdc files for lab 2 - operators, parameters, and testing
 - added the latex file for the first two courses

### CODE REVIEW

File: slides/courses/1/components.tex
Comment (aidanozo) [inline slides/courses/1/components.tex:1]: Found a typo: "iluzia unei ierarhi", it should be "ierarhii".😅

File: slides/courses/1/functional.tex
Comment (aidanozo) [inline slides/courses/1/functional.tex:1]: Another typo: "Transiztoare"

Comment (aidanozo) [reply slides/courses/1/functional.tex:1]: + "portți"

**Review by aidanozo — APPROVED** [review]
Found a few typos, otherwise it is all correct.😎

**Review by catalin-rucareanu2001 — APPROVED** [review]
I approve these changes with the caveat of changing (myself) the typos mentioned by @aidanozo

**Pull request merged**